### DISCUSSION TITLE
Region-based training in MONAI?

### DISCUSSION THREAD

**Discussion by jqmcginnis** [Project-MONAI/MONAI#6902] [Q&A]
Hello,

I recently started using MONAI for different medical imaging tasks, and really enjoy it. Thanks to the community ! :smile: 

Having worked with region-based training in nn-unet, I was wondering if something similar exists in MONAI and if someone could give me some advice with respect to its implementation. For a description of region-based training, please refer to this link below:

https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/region_based_training.md

Prior to asking this question, I took a look at the discussion forum and did some digging, however, if I missed something in the API, please let me know!

Thank you very much.

Cheers, 

Julian

**Comment by wyli** [comment]
thanks, if you are referring to the label indices mappings there's a transform to achieve this https://github.com/Project-MONAI/MONAI/blob/2daabf917a1e6afbc738d22db602f4859672a90d/monai/transforms/utility/array.py#L1201-L1208  or a more specific version in the tutorial https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/brats_segmentation_3d.ipynb
